linux/preview/SLES/README.md

```diff
@@ -1,39 +1,26 @@
 # How to build a SQL Server on SLES container image
 
-**Prerequisites - Get SLES Container Images** 
-SUSE Enterprise Linux Containers are not avalaible on any public repositories. Follow the steps below to get the container base OS locally:
+1. Ensure that host machine is running the same version of the SLES that you will build SQL Server 2019 container image, in this case we are using a host which is running SLES 12 SP 5 and building the SQL Server also on top of SLES 12 SP 5.
 
-a.	Activate the container module via yast by typing in ```yast2``` and following the prompts. 
-
-b.	Search for the available SLES images with ```zypper se *-image```
-
-c.	Install one of those images with zypper install, for example, ```zypper in sles12sp2-docker-image```
-
-d.	List out the installed images with ```sle2docker list```
-
-e.	Activate the image with sle2docker, for example ```sle2docker activate sles12sp1-docker.x86_64-1.0.5-Build10.18```
-
-f.	Check that the image available with ```docker image ls```
-
-
-For full details: https://www.suse.com/documentation/sles-12/singlehtml/book_sles_docker/book_sles_docker.html#docker.building.images
+2. Please ensure the host machine is registered using the SUSEConnect command as documented here: https://www.suse.com/support/kb/doc/?id=000018564
```

Comment:
Is there a command for the end user to verify that they successfully completed this step? 
For example is there something like a 
`SUSEConnect --get-my-SUSEConnect-status-on-this-machine`

Reply:
yup there is one, which is SUSEConnect -s, in the output you should see the something like ""status":"Registered"" with the regcode and other details. Will update the readme